### PR TITLE
Switch to Gib Potato License 🥔💃

### PR DESCRIPTION
**Pull request by chadwhitacre**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+If you feel appreciation,
+For a colleague's dedication,
+You can use this software free—
+Gib potatoes liberally!
```
Comment (cleptric) [inline LICENSE:8]: Nitpick, but with GibPotato, the plural of potato is potato 😬

Comment (chadwhitacre) [reply LICENSE:8]:
```suggestion
Gib potato liberally!
```

File: LICENSE [hunk 2]
```diff
@@ -1,16 +1,21 @@
-MIT License
+Gib Potato License
```
Comment (cleptric) [inline LICENSE:1]:
```suggestion
GibPotato License
```

**Review by cleptric — APPROVED** [review]

File: LICENSE [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
+You can copy, modify,
+Merge or publish (touch the sky!)
+Distribute, sublicense (fly!)
+Sell your copies (my oh my!)
+
+Gib us credit, though, or else
+It goes back up on the shelf.
+And in case your taters rot,
```
Comment (chadwhitacre) [inline LICENSE:17]:
```suggestion
And in case yer taters rot,
```

**Comment by cleptric** [comment]
🚀 🥔

**Pull request merged**